src/server/game/Entities/Player/Player.cpp

```diff
@@ -17139,8 +17116,8 @@ void Player::_LoadDeclinedNames(PreparedQueryResult result)
     if (!result)
         return;
 
-    delete m_declinedname;
-    m_declinedname = new DeclinedName;
+    m_declinedname.reset();
```

Comment:
eh, reset line becomes unneccessary if the next thing you do is assign new value

Reply:
does unique_ptr also destroy objects when assigning a new one? Was expecting it to only do that when going out of scope

Reply:
yes